From 4b5c2055213bf8b6d94cd1cd0f3ca798ee1df1b3 Mon Sep 17 00:00:00 2001
From: Vaibhav Bedia <vaibhav.bedia@ti.com>
Date: Wed, 14 Mar 2012 23:08:56 +0530
Subject: [PATCH 179/186] ARM: OMAP: Mailbox: Fix check for mailbox flushing

At the same time convert the pr_info to pr_debug

Signed-off-by: Vaibhav Bedia <vaibhav.bedia@ti.com>
---
 arch/arm/plat-omap/mailbox.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-omap/mailbox.c b/arch/arm/plat-omap/mailbox.c
index 940c31c..c2d550b27 100644
--- a/arch/arm/plat-omap/mailbox.c
+++ b/arch/arm/plat-omap/mailbox.c
@@ -139,8 +139,8 @@ int omap_mbox_msg_rx_flush(struct omap_mbox *mbox)
 		ret++;
 		msg = mbox_fifo_readback(mbox);
 	}
-	if (!ret)
-		pr_info("Flushed %s Rx FIFO via %d readbacks\n", mbox->name, ret);
+	if (ret)
+		pr_debug("Flushed %s Rx FIFO via %d readbacks\n", mbox->name, ret);
 
 	return ret;
 }
-- 
1.7.9.1

